core/include/kernel/interrupt.h

```diff
@@ -9,6 +9,7 @@
 #include <sys/queue.h>
 
 #define ITRF_TRIGGER_LEVEL	(1 << 0)
+#define ITRF_SHARED			(1 << 1)
```

Comment:
I see that we're not using the `BIT()` macro here. Would you mind changing these two defines to use the `BIT()` macro found in `<util.h>`?